packages/saturn/saturn.0.4.0/opam

```diff
@@ -0,0 +1,32 @@
+opam-version: "2.0"
+maintainer:"KC Sivaramakrishnan <sk826@cl.cam.ac.uk>"
+authors: ["KC Sivaramakrishnan <sk826@cl.cam.ac.uk>"]
+homepage: "https://github.com/ocaml-multicore/saturn"
+doc: "https://ocaml-multicore.github.io/saturn"
+synopsis: "Parallelism-safe data structures for multicore OCaml"
+license: "ISC"
+dev-repo: "git+https://github.com/ocaml-multicore/saturn.git"
+bug-reports: "https://github.com/ocaml-multicore/saturn/issues"
+depends: [
+  "ocaml" {>= "4.12"}
+  "dune" {>= "3.0"}
+  "domain_shims" {>= "0.1.0"}
+  "saturn_lockfree" {>= "0.4.0"}
```

Comment:
```suggestion
  "saturn_lockfree" {= version}
```
are you sure you don't prefer to lock the versions?

Reply:
Makes sense, thanks!